node/src/protocol/triple.rs

```diff
@@ -340,68 +425,70 @@ mod test {
     // Improve this before we make more similar tests
     #[test]
     fn happy_triple_generation() {
-        let mut tm = TestManagers::new(5);
-
         const M: usize = 2;
         const N: usize = M + 3;
         // Generate 5 triples
-        for _ in 0..M {
-            tm.generate(0).unwrap();
-        }
-        tm.poke_until_quiet().unwrap();
-        tm.generate(1).unwrap();
-        tm.generate(2).unwrap();
-        tm.generate(4).unwrap();
-
-        tm.poke_until_quiet().unwrap();
+        let runtime = tokio::runtime::Runtime::new().unwrap();
+        runtime.block_on(async {
```

Comment:
you can also do a `#[tokio::test]` instead on top of the function just so the code here doesn't shift into a different scope